log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java

```diff
@@ -1,6 +1,6 @@
 package kcl.teamIndexZero.traffic.log;
 
-import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
+import kcl.teamIndexZero.traffic.log.FileIO.FileInput;
```

Comment:
Es did that, as soon as I change it to small, error shows up. No clue, as to why.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: EsSent: Monday 21 March 2016 09:34To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java:
> @@ -1,6 +1,6 @@
>  package kcl.teamIndexZero.traffic.log;
>  
> -import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
> +import kcl.teamIndexZero.traffic.log.FileIO.FileInput;

@lexaux When I pulled a fresh version of the project on Kumar's laptop some of the packages you manually renamed to lowercase registered as the old uppercase ones. Weird behaviour.
@kumarawijeet try renaming "kcl.teamIndexZero.traffic.log.FileIO.FileInput;" to "kcl.teamIndexZero.traffic.log.fileIO.FileInput; and see if it's still ok.

—You are receiving this because you were mentioned.Reply to this email directly or view it on GitHub